packages/tenant-process/test/db.test.ts

```diff
@@ -78,6 +103,784 @@ describe("database test", () => {
     });
   });
   describe("readModelService", () => {
+    describe("getConsumers", () => {
+      const organizationProducerId = uuidv4();
+      const consumerId1 = uuidv4();
+      const consumerId2 = uuidv4();
+      const consumerId3 = uuidv4();
+      const mockTenant = getMockTenant();
+
+      const tenant1: Tenant = {
+        ...mockTenant,
+        id: consumerId1,
+        name: "A tenant1",
+      };
+      const tenant2: Tenant = {
+        ...mockTenant,
+        id: consumerId2,
+        name: "A tenant2",
+      };
+      const tenant3: Tenant = {
+        ...mockTenant,
+        id: consumerId3,
+        name: "A tenant3",
+      };
```

Comment:
You can delete these and use the tenants you declared in the previous route by moving them under `describe("readModelService", () => {`